mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
The way to programatically add or remove root or intermediate certs with mono requires Mono.Security.X509.  The way to do it in .Net requires System.Security.Cryptography.X509Certificates.  Sure, we *could* copy Mono.Security.dll to the windows project and eliminate the use of #if MonoCS in favor of a runtime check that causes Windows to avoid calling mono Mono.Security.X509StoreManager.CurrentUser.TrustedRoot.Import(), but we would be literally pointlessly adding the Mono.Security assembly reference to a .Net project that doesn't use it, just to eliminate the #if.